index.html

```diff
@@ -3,8 +3,24 @@
   <head>
     <title>Farm Party!</title>
     <link href="css/style.css" rel="stylesheet" type="text/css">
+    <link href='https://fonts.googleapis.com/css?family=Indie+Flower' rel='stylesheet' type='text/css'>
+    <script src="https://use.fontawesome.com/262834322c.js"></script>
   </head>
   <body>
-  	
+  <h1>Farm Party!<br><span class="fa fa-play-circle" aria-hidden="true"></span></h1>
+  <audio autoplay loop id="soundtrack">
+      <source src="media/song.mp3" type="audio/mpeg">
+    </audio>
+  	<div class="sky"></div>
+  	<div class="grass">
+  	<button class="sun">Raise the sun</button>
+  	<button class="plant">Plant the tree</button>
+  	<button class="farm">Grow the farm</button>
+    <button class="farm2">Grow more</button>
+    <button class="farm3">Grow much more</button>
+  	</div>
+  	<script src="js/jquery-3.1.0.min.js"></script>
+  	<script type="js/jquery.easing.1.3.js"></script>
```

Comment:
This should be a src attribute not type. `<script src="js/jquery.easing.1.3.js"></script>`.

This will fix your broken easing.